README.md

```diff
@@ -413,8 +413,38 @@ lines in the `~/.emacs.d/init.el` file:
 ```
 
 ## Spacemacs logo
-For Ubuntu users, follow this guide to
-[change the logo in Unity][cpaulik-unity-icon].
+
+### Ubuntu users
+
+You need to create a `.desktop` file that starts Emacs using the Spacemacs logo instead. In summary:
+
+Create a new file in `~/.local/share/applications/spacemacs.desktop` with the contents
+
+```
+[Desktop Entry]
```

Comment:
What I meant was we don't need to have this code section.

Just link to Emacs original desktop file and tell user which field to add/modify.